src/osm_configurator/model/project/configuration/category_manager.py

```diff
@@ -30,6 +32,11 @@ def __init__(self):
         Constructor of the class.
         """
         self._categories: List[Category] = []
+        self.building_category: Category = category_i.Category(BUILDING_CATEGORY_NAME)
```

Comment:
what is this? Are those default categories?  would refacctor this to another class

Reply:
Yes Simon suggested to add some default categories as described in the lastenheft. 
Refactoring into another class sounds good